### ISSUE TITLE
Resolve *.[property] enhancement

### ISSUE THREAD

**Issue by LukeHro** [JFormDesigner/FlatLaf#186]
I better describe what I want by example:
If you have *.foreground=$abc I can use getColor("Label.foreground") even if Label.foreground is not explicitly defined in the properties.
What doesn't work in this case is having something like:
MyLabel.foreground=$Label.foreground in the properties file.
I can't use MyLabel.foreground=@foreground because I want that that getColor("MyLabel.foreground") to be the value from Label.foreground if someone adds that explicitly but otherwise to be the value from *.foreground
Thanks,
Luke

**Comment by LukeHro** [comment]
PS: note that writing MyLabel.foreground=$Label.foreground in my properties file throws the following exception:
Caused by: java.lang.IllegalArgumentException: variable or property 'Label.foreground' not found
	at com.formdev.flatlaf.UIDefaultsLoader.resolveValue(UIDefaultsLoader.java:271)
	at com.formdev.flatlaf.UIDefaultsLoader.loadDefaultsFromProperties(UIDefaultsLoader.java:235)
	at com.formdev.flatlaf.UIDefaultsLoader.loadDefaultsFromProperties(UIDefaultsLoader.java:85)
If it would go past that and getColor("MyLabel.foreground") would return null or something like that I could workaround it

**Comment by DevCharly** [comment]
Which FlatLaf version do you use?

I've already changed that in FlatLaf 0.42 in commit 16977351629337a5eb029e47561d2b7d4934b26a.

Please try with latest version 0.43

**Comment by LukeHro** [comment]
Will check that out, I'm using 0.42

**Comment by DevCharly** [comment]
Yes, it should work as you described.

Looking at the line number in the stack, I seems that you're using 0.41 😉

**Comment by LukeHro** [comment]
It seems to me that we have that code from that commit (as far as I can see is also tagged "0.42") and still get the exception as I mentioned in first comment

**Comment by LukeHro** [comment]
hmm...let me double check (just noticed your last comment)

**Comment by LukeHro** [comment]
I'm pretty sure we are using .42 but you are right the line doesn't match but that's because being in early stage of what we are doing, I am using the flatlaf sources with some modifications, none in that file but the IDE reorganized the imports.

**Comment by LukeHro** [comment]
Actually it seems that it works indeed if I put MyLabel.foreground=$Label.foreground in FlatLaf.properties, FlatDark/LightLaf or other property file down the "normal" hierarchy. But we started with .36 and made a modification to load from more properties files based on the addon system but not quite and, if I put that property in such additional file, it doesn't work. The change made I kept in .42 and didn't noticed other issues. I will review that change and also check what was added regarding multiple files since .36
Thanks for everything and sorry for the misleading.
